packages/dd-trace/src/util.js

```diff
@@ -20,8 +20,50 @@ function isError (value) {
   return false
 }
 
+// Matches a glob pattern to a given subject string
+function globMatch (pattern, subject) {
```

Comment:
Is this syntax documented somewhere ? Users might think it's regex based instead

Reply:
Nope. I assume that should go in public docs though, when we get a PR for that together?

Reply:
I'm not sure how we usually do it tbh